okteto.yml

```diff
@@ -0,0 +1,8 @@
+name: hello-world
+image: okteto/python:3
+command:
+- bash
+workdir: /okteto
```

Comment:
Where are the dependencies installed?
I would install them on `workdir` to be consistent with other samples

Reply:
In python by default the dependencies are global, so they go into `/usr/local/lib/python3.8/site-packages`, but we can't mount that folder because it's also used by the system. 

We can use a virtualenv, but it adds more instructions to the getting started guide. What do you think?

Reply:
https://github.com/okteto/devenv/pull/33/files fixes it so the dependencies are installed on /okteto/venv